asset-bundle-converter/Assets/AssetBundleConverter/AssetBundleConverter.cs

```diff
@@ -343,7 +343,7 @@ private async Task ProcessAllGltfs()
 
                     if (animationMethod == AnimationMethod.Mecanim)
                     {
-                        bool isEmote = entityDTO.type.ToLower().Contains("emote");
+                        bool isEmote = entityDTO.type.ToLower().Contains("emote") || gltf.AssetPath.fileName.EndsWith("_emote.glb");

```

Comment:
Maybe this can be saved as a bool some lines above? Looks redudant and more code to maintain. I think that even the if can be removed

Reply:
wopsie, sorry, done! but the if remains 🤔